android5.0/Cheesesquare/Cheesesquare.UITests/Cheesesquare.UITests.csproj

```diff
@@ -1,13 +1,14 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project DefaultTargets="Build" ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <ProjectGuid>{9690A5F4-6D51-47C4-BB78-ACDF05B859C5}</ProjectGuid>
     <OutputType>Library</OutputType>
     <RootNamespace>Cheesesquare.UITests</RootNamespace>
     <AssemblyName>Cheesesquare.UITests</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
```

Comment:
I'd suggest using TargetFramework v4.6.1 instead.  That's what the `Xamarin.UITest Cross-Platform Test Project` template uses. See docs here on that: https://docs.microsoft.com/en-us/appcenter/test-cloud/frameworks/uitest/xamarin-forms?tabs=windows

That template also uses `ToolsVersion="4.0"` so I'd probably leave that unchanged.
Also, that template doesn't include `<TargetFrameworkProfile />`, so I'd probably omit that unless you have reason to include it.

Reply:
In fact, we just selected the recommended .NET version v4.8 in the pop-out dialog, and VS generated the code by itself.
![image](https://user-images.githubusercontent.com/11625239/154206012-fd84d711-d91f-460f-ac08-49162be2e34f.png)

We tried to upgrade to target Framework v4.6, VS still pop-up that dialog, asking to select the installed version or install v4.6. If need to install v4.5/v4.6, for this part of automation test, we need to update the corresponding task of the pipeline maybe.  

Cross-Platform uses v4.6 in Portable Class Libraries (PCL) related library projects, there is no dialog pop-up dialog ask to upgrade Framework when open project, so it didn't affect the automation test. By the way, I did find PCL template in VS2022, I'm not sure if it is deprecated, does this affect the popup upgrade dialog?
https://docs.microsoft.com/en-us/xamarin/cross-platform/app-fundamentals/pcl?tabs=windows